### PR TITLE
CTOR-1054 : Refresh cloud-aws-directconnect

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-aws-directconnect

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md [hunk 1]
```diff
@@ -5,57 +5,82 @@ title: Amazon Direct Connect
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Contenu du Pack
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md [hunk 2]
```diff
@@ -64,19 +89,16 @@ Vous trouverez plus d'informations sur la découverte d'Hôtes et son fonctionne
...
 ### Privilèges AWS
 
-Voici la liste des droits nécessaires au travers des access/secret key utilisées pour pouvoir utiliser le monitoring AWS/EC2 :
-
-| AWS Privilege                             | Description                                            |
-| :---------------------------------------- | :----------------------------------------------------- |
-| directconnect:describe-connections        | Liste l'ensemble des connexions.                       |
-| directconnect:describe-virtual-interfaces | Liste l'ensemble des interfaces virtuelles             |
-| cloudwatch:getMetricStatistics            | Récupère les métriques Cloudwatch du namespace AWS/DX. |
+Configurez un compte de service (via une combinaison d'access et secret key) et affectez-lui les privilèges suivants :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md:92]:
```suggestion
Configurez un compte de service (via une combinaison d'access key et secret key) et affectez-lui les privilèges suivants :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md [hunk 3]
```diff
@@ -169,61 +189,136 @@ dnf install centreon-plugin-Cloud-Aws-Directconnect-Api
...
-    --proxyurl='http://myproxy.mycompany.org:8080'
-    --statistic=average \
-    --timeframe='600' \
-    --period='60' \
-    --verbose
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--region='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md:301]: ?

File: pp/integrations/plugin-packs/procedures/cloud-aws-directconnect.md [hunk 4]
```diff
@@ -169,61 +190,134 @@ dnf install centreon-plugin-Cloud-Aws-Directconnect-Api
...
-    --statistic=average \
-    --timeframe='600' \
-    --period='60' \
-    --verbose
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--region='' \
+	--proxyurl=''  \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-directconnect.md:301]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md [hunk 5]
```diff
@@ -169,61 +189,136 @@ dnf install centreon-plugin-Cloud-Aws-Directconnect-Api
...
+| AWSACCESSKEY  | Set AWS access key                                                                                                                                 |                   |             |
+| AWSASSUMEROLE | Set Amazon Resource Name of the role to be assumed                                                                                                 |                   |             |
+| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option                         | awscli            |             |
+| AWSREGION     | Set the region name (required)                                                                                                                     |                   | X           |
+| AWSSECRETKEY  | Set AWS secret key                                                                                                                                 |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md:223]:
```suggestion
| AWSACCESSKEY  | Set AWS access key                                                                                                                                 |                   | X           |
| AWSASSUMEROLE | Set Amazon Resource Name of the role to be assumed                                                                                                 |                   |             |
| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option                         | awscli            |             |
| AWSREGION     | Set the region name (required)                                                                                                                     |                   | X           |
| AWSSECRETKEY  | Set AWS secret key                                                                                                                                 |                   | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md [hunk 6]
```diff
@@ -169,61 +189,136 @@ dnf install centreon-plugin-Cloud-Aws-Directconnect-Api
...
-    --aws-access-key='**********' \
-    --region='eu-west-1' \
-    --proxyurl='http://myproxy.mycompany.org:8080'
-    --statistic=average \
-    --timeframe='600' \
-    --period='60' \
-    --verbose
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md:299]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md (hunk 3 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-directconnect.md:301]:
```suggestion
	--region='eu-west-1' \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**